util/Directories.cpp

```diff
@@ -103,6 +104,27 @@ void RedirectOutputLogAndroid(int priority, const char* tag, int fd)
     });
     background.detach();
 }
+
+void CopyInitialResourceAndroid(const std::string& rel_path)
```

Comment:
```suggestion
void CopyInitialResourceAndroid(std::string_view rel_path)
```

Reply:
Then I got `error: invalid operands to binary expression ('fs::path' and 'std::string_view' (aka 'basic_string_view<char>'))`

Reply:
Maybe start to use `std::filesystem::path` instead of `boost::filesystem::path` to get support for string views?

Reply:
There might be compiler support issues still, eg. on older Apple Clang versions

Reply:
I see, Android toolchain have only Clang 9.

Reply:
@geoffthemedio Maybe merge it as is while boost doesn't supports string view for paths?

Reply:
> Android toolchain have only Clang 9.

Apple Clang and other Clang are separarte

> merge it as is

feel free...